src/core/components/button/styles.ts

```diff
@@ -135,22 +145,19 @@ export const iconXsmall = css`
 		display: block;
 		fill: currentColor;
 		position: relative;
-		width: ${size.xsmall / 2}px;
+		width: ${width.iconXsmall}px;
 		height: auto;
 	}
+	.spacer {
+		width: ${space[1]}px;
+	}
 `
-
+// TODO: fix viewBox in SVGs
 export const iconRight = css`
-	svg {
-		margin: 0 ${-size.medium / 8}px 0 ${size.medium / 4}px;
-	}
```

Comment:
We no longer need this export (or comment), so we can remove it. This will require some refactoring in [index.tsx](https://github.com/guardian/source/blob/master/src/core/components/button/index.tsx). For instance [this](https://github.com/guardian/source/blob/217fcd972e287d28381f4e588eea4d0545acad55/src/core/components/button/index.tsx#L210-L212):

```tsx
iconSvg && (!hideLabel && children)
	? iconSides[iconSide]
	: "",
```
can be simplified to

```tsx
iconSvg && (!hideLabel && children)
	? iconLeft
	: "",
```

and so on...